app/models/filters/filter_base.rb

```diff
@@ -50,7 +50,9 @@ class FilterBase < ::ModelForm
 
     validates_presence_of :start, :end
     validate do
-      errors.add(:end, 'End date must follow start date.') if start > self.end
```

Comment:
This validation should not be necessary since `ensure_dates_work` fixes incorrect date orders.

Reply:
Removed.